modules/distribution/resources/api_templates/default_ws_api_template.xml

```diff
@@ -0,0 +1,10 @@
+<api xmlns="http://ws.apache.org/ns/synapse"  name="$!apiName" context="$!apiContext">
```

Comment:
this needs to have binds-to ?

Reply:
Addressed in https://github.com/wso2/product-apim/pull/9885/commits/19ebe0528979bdc4ced4a3abf78d4186f15a41bb